### DISCUSSION TITLE
Can it style also MDI child windows?

### DISCUSSION THREAD

**Discussion by strain08** [MicaForEveryone/MicaForEveryone#484] [Q&A]
Also, some windows (toolbox style I think) get styled, others don't.
![image](https://github.com/user-attachments/assets/e84e6d8b-a9ba-4dd8-8ddb-a6352b043b76)

Is there a window class filter of some kind that can be configured?

Thank you!

**Comment by PDMADETHAT** [comment]
i really want this done

valhalla is beautiful btw..
